release/pkg/prepare_release.go

```diff
@@ -94,48 +92,52 @@ func (r *ReleaseConfig) SetRepoHeads() error {
 	return nil
 }
 
-func (r *ReleaseConfig) PrepareBundleRelease(sourceClients *SourceClients) (map[string][]Artifact, error) {
-	artifactsTable, err := r.GetBundleArtifactsData()
-	if err != nil {
-		return nil, errors.Cause(err)
+func (r *ReleaseConfig) PrepareBundleRelease(artifactsTable map[string][]Artifact, sourceClients *SourceClients) error {
+	if r.DryRun {
```

Comment:
why do we skip this in dryrun? I was expecting that we would download the artifacts and rename them but just not upload them

Reply:
I wanted the dry-run to be able to run by developers locally to test their changes to release tooling, and if we wanted to download the artifacts, they would either have to build/push all artifacts into their own AWS account, or override the AWS profile to download the prod artifacts. Both of these would entail further setup while the scope of the dry-run was only to visualize how the bundle might look after they've made changes to APIs, parsing logic etc.